x-pack/test/functional/services/transform_ui/wizard.ts

```diff
@@ -273,6 +274,12 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       await this.assertAggregationEntryExists(index, expectedLabel);
     },
 
+    async assertAdvancedPivotEditorContent(expectedValue: Record<string, any>) {
+      const advancedEditorString = await aceEditor.getValue('transformAdvancedPivotEditor');
+      const advancedEditorValue = JSON.parse(advancedEditorString);
+      expect(advancedEditorValue).to.equal(expectedValue);
```

Comment:
Currently, the tests are failing because of this one. We should use `eql` instead:
```
equal: Checks if the obj exactly equals another.
eql: Checks if the obj sortof equals another.
```

Reply:
Fixed in 7d0e3fc.